<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PJobWidget</class>
 <widget class="QWidget" name="PJobWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Resources</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeView" name="resourcesTreeView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>main.pscript</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTextEdit" name="scriptTextEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Included Applications</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>List of apps:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QListWidget" name="listOfApps">
             <property name="editTriggers">
              <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QPushButton" name="addApp">
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeApp">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>17</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Default app:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="defaultApp">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="applicationSplitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QGroupBox" name="applicationFilesBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Files</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QTreeView" name="applicationFiles">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::DragDrop</enum>
             </property>
             <property name="defaultDropAction">
              <enum>Qt::CopyAction</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="applicationInfoBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Info</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Application name:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="applicationName"/>
             </item>
             <item>
              <widget class="QPushButton" name="binariesNameAutoButton">
               <property name="text">
                <string>Auto</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Application Version:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="applicationVersion"/>
             </item>
             <item>
              <widget class="QPushButton" name="binariesVersionAutoButton">
               <property name="text">
                <string>Auto</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Executable:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLineEdit" name="applicationExecutable"/>
             </item>
             <item>
              <widget class="QPushButton" name="binariesExecutableAutoButton">
               <property name="text">
                <string>Auto</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Platform:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QComboBox" name="applicationPlatform">
               <item>
                <property name="text">
                 <string>Win32</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Win64</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>MacOSX</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Linux/Unix</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="binariesPlatformAutoButton">
               <property name="text">
                <string>Auto</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Parameters pattern:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLineEdit" name="applicationParametersPattern"/>
             </item>
             <item>
              <widget class="QPushButton" name="binariesParametersPatternAutoButton">
               <property name="text">
                <string>Auto</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="binariesAutoDetectAllButton">
             <property name="text">
              <string>Auto detect all</string>
             </property>
            </widget>
           </item>
          </layout>
          <zorder>label_3</zorder>
          <zorder>label_4</zorder>
          <zorder>label_5</zorder>
          <zorder>label_6</zorder>
          <zorder>label_7</zorder>
          <zorder>binariesAutoDetectAllButton</zorder>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite_4">
      <attribute name="title">
       <string>Parameters</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableView" name="parametersTableView"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="parametersAddButton">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="parametersRemoveButton">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite_5">
      <attribute name="title">
       <string>Results</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTreeView" name="resultsTreeView"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="addResultFileButton">
           <property name="text">
            <string>+ result file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeResultFileButton">
           <property name="text">
            <string>- result file</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addResultButton">
           <property name="text">
            <string>+ result</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeResultButton">
           <property name="text">
            <string>- result</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite_6">
      <attribute name="title">
       <string>Runs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTreeView" name="runsTreeView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
